frontend/www/js/omegaup/components/badge/List.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h2 class="panel-title">{{ title }} <span class="badge">{{ badges.length }}</span> <a class=
+      "badges-link"
+         href="/badge/list/"
+         v-if="this.forProfile">{{ this.T.wordsBadgesSeeAll }}</a></h2>
+    </div>
+    <div class="panel-body">
+      <div class="badges-container">
+        <omegaup-badge v-bind:alias="badge.badge_alias"
```

Comment:
puedes cambiar esto para mandarle el `Badge` completo en vez de en pedacitos?

```suggestion
        <omegaup-badge v-bind:badge="badge"
```